CRM/Contact/BAO/GroupNestingCache.php

```diff
@@ -69,7 +69,7 @@ static public function update() {
     }
 
     if (self::checkCyclicGraph($tree)) {
-      CRM_Core_Error::fatal(ts('We detected a cycle which we cant handle. aborting'));
+      CRM_Core_Error::fatal(ts('We detected a cycle which we can't handle. aborting'));
```

Comment:
You'll need to surround the string with double quotes or this single quote breaks it.